src/service/Searchable.java

```diff
@@ -0,0 +1,6 @@
+package service;
+
+public interface Searchable {
```

Comment:
overthink all ideas and add required changes to code

Reply:
Added changes.